docs/src/app/components/pages/components/TimePicker/Page.jsx

```diff
@@ -11,11 +11,14 @@ import TimePickerExampleSimple from './ExampleSimple';
 import timePickerExampleSimpleCode from '!raw!./ExampleSimple';
 import TimePickerExampleComplex from './ExampleComplex';
 import timePickerExampleComplexCode from '!raw!./ExampleComplex';
+import TimePickerExampleInternational from './ExampleInternational';
+import timePickerExampleInternationalCode from '!raw!./ExampleInternational';
 
 const descriptions = {
   simple: 'Time Picker supports 12 hour and 24 hour formats. In 12 hour format the AM and PM indicators toggle the ' +
   'selected time period.',
   controlled: '`TimePicker` can be used as a controlled component.',
+  localised: 'The buttons can be localised using the `wordings` property.',
```

Comment:
Somethings like:
The buttons can be localised the `cancelLabel` and `okLabel` properties.